Movie/Movie/MainScreenModule/Model/ViewData.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  ViewData.swift
+//  Movie
+//
+//  Created by Артем on 11.10.2021.
+//
+
+import UIKit
+
+enum ViewData {
```

Comment:
```suggestion
enum ViewData {
```
лучше сделай абстрактную ViewData как у Матвея